src/applications/caregivers/components/PreSubmitInfo/StatementOfTruth.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { links } from '../../definitions/content';
+
+const StatementOfTruth = ({ content }) => {
+  const { label = '', text = [] } = content;
+  return (
+    <>
+      <h3 className="vads-u-margin-top--4">{`${label} statement of truth`}</h3>
+
+      {text.map((copy, idx) => {
+        return <p key={`${label}-${idx}`}>{copy}</p>;
```

Comment:
It looks like the text within this `<p>` includes nested `<p>`s and headers. This will show up as an HTML error. Maybe change this to a div with margins?

Reply:
Where are you seeing the nested p-tags? The copy is just being pulled from an array of strings in the content file. The paragraphs are getting nested into a header tag. Don't really love that whole component setup, but thats slated for a refactor of its own in due time.